src/Core/Framework/Validation/ValidatorFactory.php

```diff
@@ -12,18 +12,22 @@
 class ValidatorFactory
 {
     /**
+     * @deprecated tag:v6.8.0 reason:new-optional-parameter - Parameter $allowExtraFields will be added and required
```

Comment:
I think we should set a default of `true`
```suggestion
     * @deprecated tag:v6.8.0 reason:new-optional-parameter - Parameter $allowExtraFields will be added
```